Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -253,7 +257,7 @@ def get_indicator_owner(indicator_value, owner=None):
 # pylint: disable=E1101
 def get_indicators(indicator_value=None, indicator_type=None, owners=None, rating_threshold=-1, confidence_threshold=-1,
                    freshness=None, associated_groups=False, associated_indicators=False, include_observations=False,
-                   include_tags=False):
+                   include_tags=False, include_attributes=False):
```

Comment:
Appended extra boolean parameter to allow for extraction of indicator attributes